Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -1870,17 +1872,9 @@ Atom *ParseV3000AtomSymbol(std::string token, unsigned int &line) {
           // according to the MDL spec, these match anything
           res->setQuery(makeAtomNullQuery());
         } else if (token == "Q") {
-          auto *q = new ATOM_OR_QUERY;
-          q->setDescription("AtomOr");
-          q->setNegation(true);
-          q->addChild(
-              QueryAtom::QUERYATOM_QUERY::CHILD_TYPE(makeAtomNumQuery(6)));
-          q->addChild(
-              QueryAtom::QUERYATOM_QUERY::CHILD_TYPE(makeAtomNumQuery(1)));
-          res->setQuery(q);
+          res->setQuery(makeQAtomQuery());
         } else if (token == "A") {
-          res->setQuery(makeAtomNumQuery(1));
-          res->getQuery()->setNegation(true);
+          res->setQuery(makeAAtomQuery());
```

Comment:
I vaguely remember X being used as A in practice as well, but I can't find any system that documents this.

Reply:
perhaps, but we don't currently parse that

Reply:
yeah, I haven't seen that.

Reply:
> I vaguely remember X being used as A in practice as well, but I can't find any system that documents this.

To wrap this up for the PR logs: after a quick slack conversation we figured this out. `X` is an abbreviation for "halogen" that's used in systems like jchem. We already support it in the CXSMILES parser and it might be worth moving support for that into the mol file parser as well. But that would be a separate PR